source/widgets/visualize/mosaicdisplay.md

```diff
@@ -6,26 +6,25 @@ Display data in a mosaic plot.
 **Inputs**
 
 - Data: input dataset
-- Data subset: subset of instances
+- Data Subset: subset of instances
 
 **Outputs**
 
-- Selected data: instances selected from the plot
+- Selected Data: instances selected from the plot
+- Data: data with an additional column showing whether an instance is selected
 
 The **Mosaic plot** is a graphical representation of a two-way frequency table or a contingency table. It is used for visualizing data from two or more qualitative variables and was introduced in 1981 by Hartigan and Kleiner and expanded and refined by Friendly in 1994. It provides the user with the means to more efficiently recognize relationships between different variables. If you wish to read up on the history of Mosaic Display, additional reading is available [here](http://www.datavis.ca/papers/moshist.pdf).
 
-![](images/Mosaic-Display-stamped.png)
+![](images/MosaicDisplay-stamped.png)
 
-1. Select the variables you wish to see plotted.
-2. Select interior coloring. You can color the interior according to class or you can use the *Pearson residual*, which is the difference between observed and fitted values, divided by an estimate of the standard deviation of the observed value. If *Compare to total* is clicked, a comparison is made to all instances.
-3. *Save image* saves the created image to your computer in a .svg or .png format.
-4. Produce a report.
+1. Select the variables you wish to see plotted. Optimize your projection with **Find Informative Mosaics**. This feature scores attributes by ReliefF and returns the top scoring variables with a simultaneous visualization update.
```

Comment:
A stylistic comment, again referring to ancient history and which is again mostly my sin: as one who has co-authored some of initial documentation, I used to write in a friendly, "chatty" style (which spread to other authors). My bad mastery of English didn't help, either. Thence "*Select the variables you wish to see plotted.*" IMHCO (C=current), we should avoid it and write "Select the variables to plot".

This continuous in the next sentence "Optimize your projection", which should, IMHCO be changed to "Optimize the projection".

In general: I would search for all appearances of `you.*` and rephrase them without mercy.